gui/qt/__init__.py

```diff
@@ -28,7 +28,7 @@
 try:
     import PyQt5
 except Exception:
-    sys.exit("Error: Could not import PyQt5 on Linux systems, you may try 'sudo apt-get install python3-pyqt5'")
+    sys.exit("Error: Could not import PyQt5, on Linux systems you may try 'sudo apt-get install python3-pyqt5'")
```

Comment:
After you commit this, I may go ahead and make this less retarded for Windows and macOS users by doing a conditional on `sys.platform`...

Reply:
Sure, you can push to the  qt_update_to_5_12 branch

Reply:
ok will do so now...